src/pages/lab-results.js

```diff
@@ -44,6 +44,16 @@ const LabResults = () => {
   });
   const isUpdate = results[0]?.lab_id ? true : false;
 
+Array.prototype.forEach.call(
+    document.querySelectorAll("input[type=text]"),
+    function (input) {
+      input.addEventListener("keyup", function () {
+        console.log("Key upped!");
+        input.value = input.value.toUpperCase();
+      });
+    }
+  );
+
```

Comment:
Same here, as mentioned for the follow ups page this code is better run in a useEffect hook

The text inputs on this page are not capitalized aswell